draft-demarco-oauth-status-attestations.md

```diff
@@ -57,21 +59,19 @@ without requiring to query any third-party entities.
 
 # Introduction
 
-Status Assertions ensure the integrity and trustworthiness of digital
+Status Assertions ensure the non-revocation of digital
```

Comment:
is there interest in suspension, revocation or other dynamic state? see https://w3c.github.io/vc-bitstring-status-list/#example-example-statuslistcredential-using-more-complex-entries

Some parties have use cases that include dynamic state that is an enumeration instead of a boolean.

Reply:
thank you, moved to an issue

Reply:
https://github.com/peppelinux/draft-demarco-oauth-status-attestations/issues/49